src/test/java/edu/ksu/canvas/attendance/submitter/CanvasAssignmentAssistantUTest.java

```diff
@@ -124,6 +129,7 @@ public void createAssignmentInCanvasNullReturnError() throws Exception {
             canvasAssignmentAssistant.createAssignmentInCanvas(COURSE_ID, attendanceAssignment, oauthToken);
             Assert.fail("Expected AttendanceAssignmentException");
         } catch (AttendanceAssignmentException e) {
+            LOG.warn("The following exception has been thrown: " + e);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "The following exception has been thrown: " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)